src/rules/linebreakStyleRule.ts

```diff
@@ -0,0 +1,49 @@
+import * as ts from "typescript";
+import * as Lint from "../lint";
+
+const OPTION_LINEBREAK_STYLE_CRLF = "CRLF";
+const OPTION_LINEBREAK_STYLE_LF = "LF";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static FAILURE_STRINGS = {
+        CRLF: `Expected linebreak to be '${OPTION_LINEBREAK_STYLE_CRLF}'`,
+        LF: `Expected linebreak to be '${OPTION_LINEBREAK_STYLE_LF}'`,
+    };
+
+    public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
+        const failures = <Lint.RuleFailure[]>[];
+        const scanner = ts.createScanner(
+            sourceFile.languageVersion,
+            false,
+            sourceFile.languageVariant,
+            sourceFile.getFullText()
+        );
+
+        // do we need to allow a way to override this option with a disabled state?
+        const linebreakStyle = this.getOptions().ruleArguments[0] || OPTION_LINEBREAK_STYLE_LF;
```

Comment:
haha nevermind, i just realized this is completely irrelevant. I am spending too much time in different linters to remember which linters do what these days...